Solved-Problems/GraphTheory/DFS.java

```diff
@@ -0,0 +1,69 @@
+import java.util.ArrayList;
+
+public class DFS {
+  // 1- --3
+  // / | \
+  // 0 | 5--6
+  // \ | /
+  // 2-- - 4
+
+  public static void main(String[] args) {
+    int vertex = 7;
+    @SuppressWarnings("unchecked") ArrayList<Edge> graph[] = new ArrayList[vertex];
+    boolean vis[] = new boolean[vertex];
+
+    createGraph(graph);
+
+    depthFirst(graph, 0, vis);
+    System.out.println();
+  }
+
+  static class Edge {
+    int src;
+    int dest;
+
+    public Edge(int s, int d) {
+      this.src = s;
+      this.dest = d;
+    }
+  }
+
+  public static void createGraph(ArrayList<Edge> graph[]) {
```

Comment:
Please fix the usage of brackets

```suggestion
  public static void createGraph(ArrayList<Edge> [] graph) {
```